pkg/gadgets/trace/network/types/types.go

```diff
@@ -18,39 +18,68 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns/ellipsis"
 	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	PktType string `json:"pktType,omitempty"`
-	Proto   string `json:"proto,omitempty"`
-	Addr    string `json:"addr,omitempty"`
-	Port    uint16 `json:"port,omitempty"`
+	PktType string `json:"pktType,omitempty" column:"type,maxWidth:9"`
+	Proto   string `json:"proto,omitempty" column:"proto,maxWidth:5"`
+	Addr    string `json:"addr,omitempty" column:"addr,template:ipaddr,hide"`
+	Port    uint16 `json:"port,omitempty" column:"ipport,template:ipport"`
 
 	/* pod, svc or other */
-	RemoteKind string `json:"remoteKind,omitempty"`
+	RemoteKind string `json:"remoteKind,omitempty" column:"kind,maxWidth:5"`
 
-	PodHostIP string            `json:"podHostIP,omitempty"`
-	PodIP     string            `json:"podIP,omitempty"`
-	PodOwner  string            `json:"podOwner,omitempty"`
-	PodLabels map[string]string `json:"podLabels,omitempty"`
+	PodHostIP string            `json:"podHostIP,omitempty" column:"podhostip,template:ipaddr,hide"`
+	PodIP     string            `json:"podIP,omitempty" column:"podip,template:ipaddr,hide"`
+	PodOwner  string            `json:"podOwner,omitempty" column:"podowner,hide"`
+	PodLabels map[string]string `json:"podLabels,omitempty" column:"padlabels,hide"`
 
 	/* if RemoteKind = svc */
-	RemoteSvcNamespace     string            `json:"remoteServiceNamespace,omitempty"`
-	RemoteSvcName          string            `json:"remoteServiceName,omitempty"`
-	RemoteSvcLabelSelector map[string]string `json:"remoteServiceLabelSelector,omitempty"`
+	RemoteSvcNamespace     string            `json:"remoteServiceNamespace,omitempty" column:"remotesvcns,template:ns,hide"`
```

Comment:
`template:ns` is used for Linux namespaces ID, and not for Kubernetes namespaces name.

cc @flyth maybe the template should be called `linuxNs` so that it is clearer?